src/main/java/org/assertj/core/annotations/Softly.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.annotations;
+
+import org.assertj.core.api.SoftAssertions;
+import org.assertj.core.extention.SoftlyExtension;
+
+import java.lang.annotation.*;
+
+
+/**
+ * Signifies that a {@link SoftAssertions} field in test
+ * class should be managed by {@link SoftlyExtension}
+ *
+ * @author Arthur Mita
+ * @see SoftlyExtension
+ */
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Softly {
```

Comment:
I'm playing with a solution that does not require this annotation, after all to find the field that is a SoftAssertions you can simply check the field type. I'll see what I can do with that approach